packages/genesys-spark-components/src/style/_typography.scss

```diff
@@ -4,73 +4,278 @@
+@mixin heading-lg-semiBold {
+  @include _heading-lg;
 
-@mixin body-font {
-  @include base-font;
+  font-weight: var(--gse-core-fontWeight-semiBold);
+}
 
-  font-size: var(--gse-core-fontSize-xs);
-  line-height: var(--gse-core-lineHeight-xs);
+@mixin heading-lg {
+  @include heading-lg-bold;
 }
 
-@mixin body-font-bold {
-  font-family: var(--gse-core-fontFamily-body), sans-serif;
+@mixin heading-md-bold {
+  @include _heading-md;
+
   font-weight: var(--gse-core-fontWeight-bold);
 }
 
-/* Headings */
+@mixin heading-md-semiBold {
+  @include _heading-md;
+
+  font-weight: var(--gse-core-fontWeight-semiBold);
+}
+
+@mixin heading-md {
+  @include heading-md-bold;
+}
+
+@mixin heading-sm-bold {
+  @include _heading-sm;
 
-@mixin h1 {
-  font-family: var(--gse-core-fontFamily-heading),
-    var(--gse-core-fontFamily-body), sans-serif;
-  font-size: var(--gse-core-fontSize-xl);
   font-weight: var(--gse-core-fontWeight-bold);
-  line-height: var(--gse-core-lineHeight-xl);
 }
 
-@mixin h2 {
-  font-family: var(--gse-core-fontFamily-heading),
-    var(--gse-core-fontFamily-body), sans-serif;
-  font-size: var(--gse-core-fontSize-lg);
+@mixin heading-sm-semiBold {
+  @include _heading-sm;
+
+  font-weight: var(--gse-core-fontWeight-semiBold);
+}
+
+@mixin heading-sm {
+  @include heading-sm-bold;
+}
+
+@mixin heading-xs-bold {
+  @include _heading-xs;
+
+  font-weight: var(--gse-core-fontWeight-bold);
+}
+
+@mixin heading-xs-semiBold {
+  @include _heading-xs;
+
   font-weight: var(--gse-core-fontWeight-semiBold);
-  line-height: var(--gse-lineHeight-md);
 }
 
-@mixin h3 {
-  font-family: var(--gse-core-fontFamily-heading),
+@mixin heading-xs {
+  @include heading-xs-bold;
+}
+
+@mixin heading-subheading-bold {
+  @include _heading-subheading;
+
+  font-weight: var(--gse-core-fontWeight-bold);
+}
+
+@mixin heading-subheading-semiBold {
+  @include _heading-subheading;
+
+  font-weight: var(--gse-core-fontWeight-semiBold);
+}
+
+@mixin heading-subheading-regular {
+  @include _heading-subheading;
+
+  font-weight: var(--gse-core-fontWeight-regular);
+}
+
+@mixin heading-subheading {
+  @include heading-subheading-bold;
+}
+
+@mixin heading-overline {
+  @include _heading-overline;
+
+  font-weight: var(
+    --gse-core-fontWeight-regular
+  ); //TODO token for this is wrong
+}
+
+/* Body Copy */
+@mixin _body-large {
+  font-family: var(--gse-core-typography-body-lg-fontFamily),
+    var(--gse-core-fontFamily-body), sans-serif;
+  font-size: var(--gse-core-typography-body-lg-fontSize);
+  line-height: var(--gse-core-typography-body-lg-lineHeight);
+}
+
+@mixin _body-medium {
+  font-family: var(--gse-core-typography-body-md-fontFamily),
     var(--gse-core-fontFamily-body), sans-serif;
-  font-size: var(--gse-core-fontSize-md);
+  font-size: var(--gse-core-typography-body-md-fontSize);
+  line-height: var(--gse-core-typography-body-md-lineHeight);
+}
+
+@mixin _body-small {
+  font-family: var(--gse-core-typography-body-sm-fontFamily),
```

Comment:
Very minor, do you think these would be better prefixed with `lg` , `md` , `sm` for consistency.

Reply:
Done